languages/ruby/exercises/concept/booleans/.meta/example.rb

```diff
@@ -0,0 +1,15 @@
+class RidePass
+  MINIMUM_HEIGHT = 120
+
+  def issue(height)
+    rand(100) if height > MINIMUM_HEIGHT
+  end
+
+  def valid?(pass)
+    !pass.nil? && pass != false
+  end
+
+  def revoke_pass(pass)
+    false unless pass.nil?
+  end
```

Comment:
This is temporary, as the `false` is not being stored to the pass.

Reply:
Agreed, the example as a whole is a bit contrived and simplified, but I intend to describe in the instructions that the ride pass has 3 states:

- non-existence - a `nil` value
- valid - the `rand(1000)` value
- revoked - the `false` value

The last one is mostly to just give an opportunity to use false and some false expressions.

Reply:
Then this method seems to be a query.  So `revoke_pass?` but the communication is wrong in this regard.

Reply:
The name would then be `revoke?` because the pass is the subject, and assumed.

Reply:
```suggestion
  def revoke(pass)
    false unless pass.nil?
  end
```

I want the message to say "revoke this pass" and the action taken to be "it is now revoked (false) unless it didn't exist to start"